swarm/network/stream/stream.go

```diff
@@ -84,6 +83,15 @@ func NewRegistry(addr *network.BzzAddr, delivery *Delivery, db *storage.DBAPI, i
 	})
 	RegisterSwarmSyncerServer(streamer, db)
 	RegisterSwarmSyncerClient(streamer, db)
```

Comment:
Here we do `RegisterSwarmSyncerServer` and Client - do we actually also need to do it in `common_test.go` line 88?

Reply:
Yes, you are right. Creating a new PR.